test/test_server_selection.py

```diff
@@ -85,7 +85,9 @@ def all_hosts_started():
             )
 
         wait_until(all_hosts_started, "receive heartbeat from all hosts")
-        expected_port = max([n.address[1] for n in client._topology._description.readable_servers])
+        expected_port = max(
+            [n.address[1] for n in client._topology._description.readable_servers]
+        )  # type:ignore[type-var]
```

Comment:
This needs to be moved because of auto-formatting

Reply:
The formatter still mangled it, you'll need to move the comment up to the line with `n.address[1]`